src/run_lemon_bot.py

```diff
@@ -316,6 +316,52 @@ def valid(query):
         return
     await client.change_presence(game=discord.Game(name=input))
 
+def pos_in_string(string, arg):
+    return string.find(arg)
+
+async def cmd_add_censored_word(client, message, input):
+    perms = message.channel.permissions_for(message.author)
+    if not perms.administrator:
+        await client.send_message(message.channel, 'You do not have permissions for this command.')
+        return
+    if not input or (not input[0:6].startswith('words=')):
+        await client.send_message(message.channel, 'Usage: !addblockedword **words**=word1, word2, word3, word4 '
+                                                   '**exchannel**=Main **message**=Profanity is not allowed.\n '
+                                                   'If no channel or message is specified, no channels will be '
+                                                   'excluded and default info message will be used.')
+        return
+    bannedwords, exchannel, infomessage = parse_blocked_word_message(input)
+    if not bannedwords:
+        await client.send_message(message.channel, "You must specify words to ban.")
+        return
+    if (pos_in_string(input, "message=") > pos_in_string(input, "exchannel=")) and ("message=" and "exchannel=" in input):
```

Comment:
`"message=" and "exchannel=" in input` ei varmaan toimi niinku olettais kun `"message="` on aina true eli ehto on käytännössä pelkkä `"exchannel=" in input`. Tarkotitko `"message=" in input and "exchannel=" in input`?

Ehto kokonaisuudessaan kyl taitaa sattumalta toimia nykyiselläänkin ton `pos_in_string(input, "message=") > pos_in_string(input, "exchannel=")` kanssa.